hackathon_site/dashboard/frontend/src/components/dashboard/EditTeam/EditTeam.tsx

```diff
@@ -109,8 +138,15 @@ export const EditTeam = ({ teamCode, canChangeTeam, teamSize }: TeamModalProps)
                     <TeamChangeForm
                         teamCode={teamCode}
                         canChangeTeam={canChangeTeam}
+                        canLeaveTeam={false}
                         teamSize={teamSize}
                     />
+                    {isJoinTeamLoading && (
```

Comment:
Instead of the loading bar being here, I think it would make more sense to put a circular loading icon on the submit or leave team buttons since that is where the action is initiated

Reply:
I would say I prefer the progress bar, here are the reasons:
1. The MUI version we are using currently is V4, which doesn't have a component called `<LoadingButton>`. Which means we need to write our own CSS to achieve the loading animation. On MUI's document, they give a demo, which looks really ugly from my perspective.
<img width="165" alt="CleanShot 2022-08-21 at 16 44 14@2x" src="https://user-images.githubusercontent.com/36619969/185810161-84579ed0-3710-47d7-a168-bc2ed7d9e460.png">

2. I love the animation 👻